server/service/src/requisition/request_requisition/insert_program.rs

```diff
@@ -204,12 +204,17 @@ fn generate(
         Some(ProgramIndicatorFilter::new().program_id(EqualFilter::equal_to(&program.id))),
     )?;
 
+    let customer_name_id = StoreRowRepository::new(connection)
+        .find_one_by_id(&ctx.user_id.clone())?
+        .ok_or(RepositoryError::NotFound)?
+        .name_link_id;
+
```

Comment:
To ensure merge safety, I'm not entirely clear on if we should be using name_link_id or querying the store repository to find the name_row.id.